### ISSUE TITLE
Rusty Hatch Missing Outer-lining Texture (Invisible)

### ISSUE THREAD

**Issue by ghost** [Ordinastie/MalisisDoors#331]
Current Forge Version In-use: 14.23.5.2779
Current Mod Version In-use: malisisdoors-1.12.2-7.3.0

https://imgur.com/a/A378sMF

**Comment by Austinsmc** [comment]
I get the same problem and with the door maker and block maker
in 1.12.2-14.23.5.2847

**Comment by nalilord** [comment]
Yep, having the same problem. Also a block update near the hatch will cause it to render correctly again for some time, but it will break again eventually.

**Comment by wllp** [comment]
I also have this issue.
